docs/user-documentation/content_types.md

```diff
@@ -28,7 +30,7 @@ There are multiple tabs with different options to configure your Content Type:
 - _Manage form display_: Set the order in which fields appear in a form, including nesting; set how the user will enter data into a field (i.e., text field, drop-down list, radio butons, etc); set fields to be hidden in the display
```

Comment:
There's a typo here "butons" and I suggest "set fields to be hidden in the display" be changed to "set fields to be hidden in the form."